### PR TITLE
Add description of VIRCADIA_LOG_OPTIONS

### PR DESCRIPTION
**Pull request by daleglass**

### CODE REVIEW

**Comment by daleglass** [comment]
This is documentation for kasenvr/project-athena#804

**Comment by kasenvr** [comment]
Looks good to merge once the other is merged.

File: docs/source/developer/interface/environment-variables.md
Comment (JulianGro) [inline docs/source/developer/interface/environment-variables.md:19]:
```suggestion
Logging to the console can be controlled with the `VIRCADIA_LOG_OPTIONS` variable. It can be set to any combination of the following:
```
`Logging on the console` sounds weird to me. Normally `on` would refer to the physical machine or furniture. I would suggest `Logging to the console` as Vircadia is sending messages to the standard output.

**Comment by kasenvr** [comment]
Well, there is one thing that I can think of... millisecond precision means what exactly? I'm mildly confused, does that mean how detailed the timestamp will be? What is it if millisecond precision is off? Might be nice to specify in a few words.

**Comment by JulianGro** [comment]
> Well, there is one thing that I can think of... millisecond precision means what exactly? I'm mildly confused, does that mean how detailed the timestamp will be? What is it if millisecond precision is off? Might be nice to specify in a few words.

Currently it goes up to centiseconds. E.g: `[10/15 05:06:02]`
Milliseconds would mean something like `[10/15 05:06:021]`

**Comment by daleglass** [comment]
> > Well, there is one thing that I can think of... millisecond precision means what exactly? I'm mildly confused, does that mean how detailed the timestamp will be? What is it if millisecond precision is off? Might be nice to specify in a few words.
> 
> Currently it goes up to seconds. E.g: `[10/15 05:06:02]`
> Milliseconds would mean something like `[10/15 05:06:02:121]`

Yup. It'll say `[10/15 05:06:02.121]` to be more exact.

**Review by JulianGro — APPROVED** [review]

File: docs/source/developer/interface/environment-variables.md
Comment (kasenvr) [inline docs/source/developer/interface/environment-variables.md:25]:
```suggestion
Will result in logging with millisecond precision, and color coding messages by severity.

    VIRCADIA_LOG_OPTIONS=milliseconds

Will result in only millisecond precision being utilized.
```

**Review by kasenvr — CHANGES_REQUESTED** [review]

**Comment by kasenvr** [comment]
I'll make the revisions on my PR since it already updates formatting and is based on this. Thanks!

**Pull request merged**
